anvil/src/udev.rs

```diff
@@ -226,7 +223,18 @@ pub fn run_udev(
     let session_event_source = auto_session_bind(notifier, event_loop.handle())
         .map_err(|(e, _)| e)
         .unwrap();
-    let udev_event_source = udev_backend_bind(udev_backend, &event_loop.handle())
+
+    for (dev, path) in udev_backend.device_list() {
+        udev_handler.device_added(dev, path.into())
+    }
+
+    let udev_event_source = event_loop
+        .handle()
+        .insert_source(udev_backend, move |event, _, _state| match event {
+            UdevEvent::Added { device_id, path } => udev_handler.device_added(device_id, path),
+            UdevEvent::Changed { device_id } => udev_handler.device_changed(device_id),
+            UdevEvent::Removed { device_id } => udev_handler.device_removed(device_id),
+        })
```

Comment:
I really like, that the `UdevHandler` is now captured by a closure instead of being completely gone into the backend.
This style should be much more flexible to use. :+1: